mmv1/products/cloudrunv2/Service.yaml

```diff
@@ -899,6 +899,15 @@ properties:
             name: 'mesh'
             description: |-
               The Mesh resource name. For more information see https://cloud.google.com/service-mesh/docs/reference/network-services/rest/v1/projects.locations.meshes#resource:-mesh.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'nodeSelector'
+        min_version: beta
+        description: Node Selector describes the hardware requirements of the resources.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'accelerator'
```

Comment:
Can we make this field required, given this is the only subfield under `node_selector` currently. We'd generally want to prevent users sending empty blocks.

Reply:
If different fields are added later and all are optional, is it a breaking change to remove required on `accelerator`?

Reply:
Nope, required -> optional is non-breaking